internal/kubenurse/server.go

```diff
@@ -122,7 +122,8 @@ func New(ctx context.Context, c client.Client) (*Server, error) { //nolint:funle
 			bucket, e := strconv.ParseFloat(bucketStr, 64)
 
 			if e != nil {
-				log.Fatalf("couldn't parse one of the custom histogram buckets. error:\n%v", e)
+				slog.Error("couldn't parse one of the custom histogram buckets", "bucket", bucket, "err", e)
+				return nil, e
 			}
```

Comment:
should we exit here. if histogram is not parsable? earlier `log.Fatal`  was exiting here...

Reply:
yeah, good point.